NuGet.config

```diff
@@ -4,5 +4,7 @@
     <clear />
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="HttpSysLocal" value="C:\Users\jukotali\code\httpsysserver\artifacts\build" />
```

Comment:
You can't check this in.

Reply:
Yeah I had to wait for the package to be available.